packages/bson-bench/.eslintrc.json

```diff
@@ -0,0 +1,296 @@
+      },
+      {
+        "object": "context",
+        "property": "only"
+      }
+    ],
+    "no-restricted-globals": [
+      "error",
+      {
+        "name": "clearTimeout",
+        "message": "Use `import { clearTimeout } from 'timers';` instead"
+      },
+      {
+        "name": "clearImmediate",
+        "message": "Use `import { clearImmediate } from 'timers';` instead"
+      },
+      {
+        "name": "clearInterval",
+        "message": "Use `import { clearInterval } from 'timers';` instead"
+      },
+      {
+        "name": "setTimeout",
+        "message": "Use `import { setTimeout } from 'timers';` instead"
+      },
+      {
+        "name": "setImmediate",
+        "message": "Use `import { setImmediate } from 'timers';` instead"
+      },
+      {
+        "name": "setInterval",
+        "message": "Use `import { setInterval } from 'timers';` instead"
+      }
+    ],
+    "prettier/prettier": "error",
+    "tsdoc/syntax": "warn",
+    "valid-typeof": "error",
+    "eqeqeq": [
+      "error",
+      "always",
+      {
+        "null": "ignore"
+      }
+    ],
+    "simple-import-sort/imports": "error",
+    "simple-import-sort/exports": "error",
+    "import/first": "error",
+    "import/newline-after-import": "error",
+    "import/no-duplicates": "error",
+    "strict": [
+      "error",
+      "global"
+    ],
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/require-await": "off",
+    "@typescript-eslint/consistent-type-imports": [
+      "error",
+      {
+        "prefer": "type-imports",
+        "disallowTypeAnnotations": false,
+        "fixStyle": "inline-type-imports"
+      }
+    ],
+    "no-restricted-imports": [
+      "error",
+      {
+        "paths": [
+          {
+            "name": ".",
+            "message": "Please import directly from the relevant file instead."
+          },
+          {
+            "name": "..",
+            "message": "Please import directly from the relevant file instead."
+          }
+        ]
+      }
+    ],
+    "no-restricted-syntax": [
+      "error",
+      {
+        "selector": "TSEnumDeclaration",
+        "message": "Do not declare enums"
+      },
+      {
+        "selector": "BinaryExpression[operator=/[=!]==/] Identifier[name='undefined']",
+        "message": "Do not strictly check undefined"
+      },
+      {
+        "selector": "BinaryExpression[operator=/[=!]==/] Literal[raw='null']",
+        "message": "Do not strictly check null"
+      },
+      {
+        "selector": "BinaryExpression[operator=/[=!]==?/] Literal[value='undefined']",
+        "message": "Do not strictly check typeof undefined (NOTE: currently this rule only detects the usage of 'undefined' string literal so this could be a misfire)"
+      }
+    ],
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "argsIgnorePattern": "^_"
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": [
+        "*.d.ts"
```

Comment:
There are overrides in here that don't apply, can we trim this down? we've also put a bunch of rule exemptions in for the tech debt that are in `test/**/*.js` / `"test/**/*.ts"` in the driver, we don't have that debt here so we don't need to turn off a majority of the rules that have be disabled.